Solutions/GoogleCloudPlatformIAM/SolutionMetadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-gcpiam",
+	"firstPublishDate": "2021-07-30",
+	"providers": ["Google"],
+	"categories": {
+		"domains" : ["Cloud Provider","Identity"],
+		"verticals": []
+	},
+	"support": {
+		"name": "Microsoft Corporation",
+		"email": "support@microsoft.com",
+		"tier":"Microsoft",
+		"link":"https://support.microsoft.com"
+	}
+}
+
```

Comment:
Hey @v-mchatla, please remove blank rows

Reply:
As suggested, removed the extra space